spk/plexpy-custom/src/installer.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/sh
+
+# Package
+PACKAGE="plexpy-custom"
+DNAME="PlexPy Custom"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+SSS="/var/packages/${PACKAGE}/scripts/start-stop-status"
+PYTHON_DIR="/usr/local/python"
+GIT_DIR="/usr/local/git"
+PATH="${INSTALL_DIR}/bin:${INSTALL_DIR}/env/bin:${PYTHON_DIR}/bin:${GIT_DIR}/bin:${PATH}"
+USER="plexpy-custom"
+GROUP="nobody"
+CFG_FILE="config.ini"
+GIT="${GIT_DIR}/bin/git"
+VIRTUALENV="${PYTHON_DIR}/bin/virtualenv"
+TMP_DIR="${SYNOPKG_PKGDEST}/../../@tmp"
+
+SERVICETOOL="/usr/syno/bin/servicetool"
+FWPORTS="/var/packages/${PACKAGE}/scripts/${PACKAGE}.sc"
+
+preinst ()
+{
+    # Check fork
+    if [ "${SYNOPKG_PKG_STATUS}" == "INSTALL" ] && ! ${GIT} ls-remote --heads --exit-code ${wizard_fork_url:=git://github.com/drzoidberg33/plexpy/plxpy.git} ${wizard_fork_branch:=master} > /dev/null 2>&1; then
+        echo "Incorrect fork"
+        exit 1
+    fi
+
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+
+    # Create a Python virtualenv
+    ${VIRTUALENV} --system-site-packages ${INSTALL_DIR}/env > /dev/null
+
+    if [ "${SYNOPKG_PKG_STATUS}" == "INSTALL" ]; then
+        # Clone the repository
+        ${GIT} clone --depth 10 --recursive -q -b ${wizard_fork_branch:=master} ${wizard_fork_url:=git://github.com/drzoidberg33/plexpy.git} ${INSTALL_DIR}/var/plexpy > /dev/null 2>&1
+    fi
+
+    # Create user
+    adduser -h ${INSTALL_DIR}/var -g "${DNAME} User" -G ${GROUP} -s /bin/sh -S -D ${USER}
+    
+    # get IP
+    sIPNAS=`/usr/syno/sbin/synonet --show | grep -m 1 IP:  | awk -F: '{gsub(/[ \t]+/, "", $2); print $2}'`
+    
+    # Add Host IP to config.ini
+    echo "http_host = ${sIPNAS}" >> ${INSTALL_DIR}/var/{CFG_FILE}
```

Comment:
@Dr-Bean - I added this as there was an issue when PlexPy ran without an IP and a config.ini file that already included an assigned port. I'll double check why I did this, or if no longer required, remove the detection and assigning of the IP address. I do need to keep a config.ini with the `web_port = 8010`, or the default installation will assign a default port (8183 from memory).